Makefile

```diff
@@ -132,7 +146,7 @@ $(MESOS_SPARK_TEST_JAR_PATH): mesos-spark-integration-tests
 	cp -v test-runner/target/scala-2.11/mesos-spark-integration-tests-assembly-0.1.0.jar $@
 
 # Special directive to allow building the test jars separately from running the tests.
-test-jars: $(DCOS_SPARK_TEST_JAR_PATH) $(MESOS_SPARK_TEST_JAR_PATH)
+test-jars: $(DCOS_SPARK_TEST_JAR_PATH)
```

Comment:
It's a bit of a long story but basically we hadn't been running those tests since the last release in March. When we found the error in CI, we opted to turn off the test bc the project's source code didn't appear actively supported (other than our own devs fixing it). Mahendra recommended not including someone else's test code as part of our own so really we've just been building the jar and not using it. This PR just makes it explicit.

I left it as part of the other targets in case we decide we want to explore this again in the future but i can remove completely if it's more confusing than helpful.